docs/src/release-notes-js.md

```diff
@@ -9,7 +9,7 @@ title: "Release notes"
 
 #### ⚡️ Introducing Playwright TestRunner
 
-[Playwright TestRunner](./test-intro.md) is a **new test runner** built from scratch by Playwright team specifically to accommodate end-to-end testing needs:
+[Playwright TestRunner](./intro.md) is a **new test runner** built from scratch by Playwright team specifically to accommodate end-to-end testing needs:
```

Comment:
not related to this PR: sometimes we use test-runner, sometimes TestRunnner and mostly Playwright Test afaik. Can we stick with Playwright or clarify for a standardised variant?